docs/pages/repo/docs/core-concepts/monorepos/_meta.json

```diff
@@ -1,6 +1,7 @@
 {
   "running-tasks": "Running Tasks",
   "task-dependencies": "Task Dependencies",
+  "task-graph": "The Task Graph",
```

Comment:
stumbled across this, need to check out if it's worth resurrecting https://github.com/vercel/turbo/pull/3973
cc @nathanhammond @anthonyshew

Reply:
That's for building visualizations with react-flow, not necessarily a doc.

I have plans (currently only in my head because still ideating) to incorporate a graph-building system for our docs based on this idea. (e.g. No more Figma, just write it in mermaid.js/react-flow)